homeassistant/components/sensor/recorder.py

```diff
@@ -76,8 +90,13 @@
 # Keep track of entities for which a warning about unsupported unit has been logged
 WARN_UNSUPPORTED_UNIT: HassKey[set[str]] = HassKey(f"{DOMAIN}_warn_unsupported_unit")
 WARN_UNSTABLE_UNIT: HassKey[set[str]] = HassKey(f"{DOMAIN}_warn_unstable_unit")
+# Keep track of entities for which a warning about statistics mean algorithm change has been logged
+WARN_STATISTICS_MEAN_CHANGED: HassKey[set[str]] = HassKey(
+    f"{DOMAIN}_warn_statistics_mean_change"
+)
 # Link to dev statistics where issues around LTS can be fixed
 LINK_DEV_STATISTICS = "https://my.home-assistant.io/redirect/developer_statistics"
+MEAN_TYPE_CHANGED_ISSUE = "mean_type_changed"
```

Comment:
It's weird that one of the four supported issues is using a constant and the others just hard coded strings.
Can we please keep the same pattern for all? Maybe a small cleanup-PR adding constants for the existing issues before we merge this?

Reply:
https://github.com/home-assistant/core/pull/141441